### PR TITLE
Use scripts instead of bin for Windows

### PR DESCRIPTION
**Pull request by kalbermattenm**
Just some changes in CONST_CHANGELOG.txt_tmpl (in order for me not to waste everybody's time...)

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -161,6 +161,8 @@ You should also add a Layer group named 'background' with the background layers.
     .build/venv/bin/alembic --config alembic.ini upgrade 7d271f4527cd
     .build/venv/bin/treeitem_uniquename
 
+    If you are using Windows, then please replace `bin` with `scripts` in the paths
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: What about adding the `.cmd`?

**Comment by kalbermattenm** [comment]
Actually, it is a Windows executable (`alembic.exe`), thus you do not need to write it using the file extension.

In Windows, running
`> alembic.exe`
or running
`> alembic`
is the same...

**Comment by sbrunner** [comment]
And `make -f <user>.mk .build/venv/scripts/alembic` will also works?

**Comment by kalbermattenm** [comment]
Yes it will

**Comment by sbrunner** [comment]
@kalbermattenm thanks :-)

**Pull request merged**